### ISSUE TITLE
Ipad IOS 9.1 resuming safari issues

### ISSUE THREAD

**Issue by nani3105** [videojs/video.js#2815]
Repo Steps:
1. Go to the sample test page (http://docs.videojs.com/docs/examples/simple-embed/index.html) on ipad safari.
2. Play the video.
3. Press the home button. 
   4.Wait for a minute or so,  Open safari again.

Expected : The video pauses when safari goes into the background. When opening safari again, the video should resume playing again.

Actual: The player just plays the audio and displays the poster.

**Comment by richardbushell** [comment]
Yes, latest Safari on Apple devices doesn't work properly, even on Mac Desktop.
Reported several times here:
https://github.com/videojs/videojs-contrib-dash/issues/22
https://github.com/videojs/video.js/issues/2807
Seems there's a problem between Safari and VJS if there's a POSTER included.
We are PC based, but when we have had a chance to test on latest Mac we also have problems with the video not playing initially or replaying at end (spinning disc only), as well as the same poster issue with audio only after seeking.
Hopefully, someone can figure this out asap.

**Comment by jemoreno** [comment]
I have gotten autoplay to work just fine in Safari on the desktop and the iPad: https://tv.dcccd.edu/embed/?live=DCTV&autoplay

**Comment by brandoncc** [comment]
I can confirm this issue still exists.

**Comment by tmm1** [comment]
I'm seeing a similar issue on desktop Safari as well, where the poster will sometimes appear after a seek.

**Comment by gkatsev** [comment]
There's been a lot of Video.js releases since here and several iOS versions as well. Seems like it's not an issue with latest Video.js and iOS 10. Closing this. If it's still an issue please open a new issue. Thanks.
